package.json

```diff
@@ -11,10 +11,13 @@
   "scripts": {
     "api-gen": "./hack/gen-api-docs.sh",
     "build": "antora generate --clean --fetch antora-playbook.yml",
-    "preview:netlify": "netlify deploy --dir public"
+    "preview:netlify": "netlify deploy --dir public",
+    "lint:docs": "./hack/vale-check.sh",
+    "lint:docs:fix": "./hack/vale-check.sh",
+    "lint:docs:strict": "./hack/vale-check.sh --minAlertLevel=error",
```

Comment:
This sounds like opposite of strict check, at least I'm used to `strict=warnings are errors`

Reply:
I think that there is also a benefit to only showing the errors as that is what will block PRs. Is there a better name that we can have to this script target?

Reply:
I changed this to `lint:docs:errors`